first_hour_on_vwb/creating_a_data_collection.ipynb

```diff
@@ -0,0 +1,735 @@
+    "            createWorkspaceCommandList = [\n",
+    "                \"terra\", \"workspace\", \"create\",\n",
+    "                f\"--id={self.input_workspace_id.value.strip()}\",\n",
+    "                f\"--description=\\\"{self.input_description.value}\\\"\",\n",
+    "                f\"--name={self.input_name.value.strip()}\",\n",
+    "            ]\n",
+    "            print('Running command to create workspace...')\n",
+    "            print('\\n'.join(createWorkspaceCommandList))\n",
+    "            print('')\n",
+    "            print(\"Your workspace will be ready in less than one minute...\")\n",
+    "            result = subprocess.run(createWorkspaceCommandList, capture_output = True, text = True, timeout=180, check=True)\n",
+    "            print(result.stderr) if not result.stdout else print(result.stdout)\n",
+    "\n",
+    "create_ws_widget = CreateWorkspaceWidget()\n",
+    "display(create_ws_widget.vb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Convert new workspace to data collection\n",
+    "<a id=\"convert-to-dc\"></a>\n",
+    "\n",
+    "Now you'll convert your newly created workspace, to which you have added resources, into a data collection which can be shared with others and added to other workspaces. \n",
+    "Run the cell below to create a widget, then populate the widget's input fields and click the button to convert the workspace to a data collection. Please note that until you <a href=\"#publish-version\">publish a version</a> in the next section, your data collection will not appear in the data catalog.\n",
+    "\n",
+    "Widget input parameters include:\n",
+    "- `workspace_id`: Must match the workspace ID of the workspace created above.\n",
+    "- `short-description`: Must be a string. This description will be visible in the Add a Data Collection modal and should summarize the purpose and/or contents of your data collection.\n",
+    "\n",
+    "The output should resemble:\n",
+    "```\n",
+    "Workspace properties successfully updated.\n",
+    "ID:                <STUDY_NAME>_<YYMMDD>_dc_ws\n",
+    "Name:               <STUDY_NAME>-Data-<YYMMDD>\n",
+    "Description:       <DESCRIPTION>\n",
+    "Cloud Platform:    GCP\n",
+    "Google project:    <GOOGLE_PROJECT_ID>\n",
+    "Cloud console:     https://console.cloud.google.com/home/dashboard?project=<GOOGLE_PROJECT_ID>\n",
+    "Properties:\n",
+    "  terra-workspace-short-description: <DESCRIPTION>\n",
+    "  terra-type: data-collection\n",
+    "Created:           YYYY-MM-DD\n",
+    "Last updated:      YYYY-MM-DD\n",
+    "# Resources:       <NUMBER_OF_RESOURCES>\n",
+    "Workspace properties successfully updated.\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "class ConvertToDataCollectionWidget(object):\n",
```

Comment:
When I run this widget, it now throws an error for me: https://screenshot.googleplex.com/BHE6VgKR2ME47se